app/packages/looker/src/elements/util.ts

```
@@ -105,7 +105,8 @@ export const getClampedTime = (
 };
 
 export const getTime = (frameNumber: number, frameRate: number): number => {
-  return (frameNumber - 1 + 0.01) / frameRate;
+  const time = (frameNumber - 1 + 0.01) / frameRate;
+  return isFinite(time) ? time : 0.0;
```

Biome

[error] isFinite is unsafe. It attempts a type coercion. Use Number.isFinite instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isFinite instead.


(lint/suspicious/noGlobalIsFinite)